charts/backbeat/values.yaml

```diff
@@ -1,12 +1,25 @@
 # Default values for backbeat.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+orbit:
+  enabled: true
```

Comment:
Can you expose this to the zenko/values.yaml as well? You will have to make a backbeat section of the zenko ```values.yaml``` file

Reply:
There's no need to 'expose' this in `zenko/values.yaml`. All dependencies' `values` are available as-is.

Reply:
While yes they are all available, an advantage of having certain values in zenko/values.yaml is so that users can easily identify the most relevant settings of how Zenko is being deployed. Which is why I was asking for it to be 'exposed' so that it can easily be identified that not only cloudserver-front has Orbit management but backbeat as well for anyone wanting to make their own changes. 

Not a make or break but just a suggestion.

Reply:
I agree, the actual configuration of enabling/disabling Orbit should be a global not per component.